### ISSUE TITLE
[IOTS-525] Cannot login after configuring email address as the username

### ISSUE THREAD

**Issue by madhawap** [wso2/product-iots#609]
Steps followed: 1. Configured email address as the username according to &#91;1&#93;.  2. Created a user with user's username in an email address format (e.g. username: kimwso2@gmail.com) 3. Tried to set the password through devicemgt app which failed. Then I tried to set the password through the mgt console. The password was changed successfully and login using the email address as the username is successful when attempting to login to the mgt console. However, login with email address and password to the devicemgt app still fails.  Please check why login to devicemgt fails and why the password cannot be set through the devicemgt app for a user with email address as the username.  &#91;1&#93; https://docs.wso2.com/display/IS530/Using+Email+Address+as+the+Username<p>Reference: <a href='https://wso2.org/jira/browse/IOTS-525'>https://wso2.org/jira/browse/IOTS-525</a></p>

**Comment by sherenem** [comment]
Tried this out again using 3.1.0 RC2 pack. I can successfully login to the devicemgt app using email address as the username now even when using the default JDBC user store. 

Still can't set the password through the devicemgt app but since this can be done using mgtconsole, this is only a minor problem. 

I updated the documentation: https://docs.wso2.com/display/IoTS310/Using+Email+Address+as+the+Username
